src/main/scala/GeographicConverter.scala

```diff
@@ -0,0 +1,129 @@
+      println("  <j2g/g2j> latitude longitude")
+      println("  'j2g' means converting latitude and longitude from Japanese geographic coordinates to Global geographic coordinates.")
+      println("Case2. convert degree")
+      println("  <dms2deg/deg2dms> degree")
+      println("  'dms2deg' means converting degree from DMS format (139/41/30.3) to DEG format (139.691749).")
+    }
+  }
+
+  /**
+    * convert Geographic Coordinates
+    *
+    * @param params
+    * @return String
+    */
+  def convertCoordinates(params: Array[String]): Map[String, Double] = {
+    var result: Array[Double] = new Array[Double](2)
+    val convertUrl = buildConverterUrl(params)
+    val json = Json.parse(scala.io.Source.fromURL(convertUrl).mkString)
+    val a = (json \ "OutputData" \ "latitude").get.asInstanceOf[JsString].value
+    val b = (json \ "OutputData" \ "longitude").get.asInstanceOf[JsString].value
+    Map("latitude" -> a.toDouble, "longitude" -> b.toDouble)
+  }
+
+  /**
+    * build Convert Coordinates API URL
+    *
+    * @param args
+    * @return String
+    */
+  def buildConverterUrl(args: Array[String]): String = {
+    val baseUrl: String = "http://vldb.gsi.go.jp/sokuchi/surveycalc/tky2jgd/tky2jgd.pl?"
+    var params: Map[String, String] = Map("outputType" -> "json", "sokuti" -> "1", "Place" -> "1")
+
+    if (args.length > 0 && args(0) == "g2j") {
+      params += ("sokuti" -> "2")
+    }
+    if (args.length > 1) {
+      params += ("latitude" -> args(1))
+    }
+    if (args.length > 2) {
+      params += ("longitude" -> args(2))
+    }
+    val paramsToString = params.map { case (key, value) => s"${key}=${value}" }.mkString("&")
+
+    baseUrl + paramsToString
+  }
+
+  /**
+    * Convert Degree Format (DEG/DMS)
+    *
+    * @param args
+    * @return
+    */
+  def convertDegree(args: Array[String]): Map[String, String] = {
+    val convertMode: String = args(0)
+    val baseDegree: String = args(1)
+
+    var result: Map[String, String] = Map("DMS" -> baseDegree, "DEG" -> baseDegree)
+    if (args.length > 0 && convertMode == "dms2deg") {
+      var convertResult: String = convertDmsToDeg(baseDegree)
+      result += ("DMS" -> convertResult)
+    } else if (args.length > 0 && convertMode == "deg2dms") {
+      var convertResult: String = convertDegToDms(baseDegree)
+      result += ("DEG" -> convertResult)
+    }
+
+    result
+  }
+
+  /**
+    * Convert Degree DEG format(139.691749) to DMS format (139/41/30.3)
+    *
+    * @param baseDegree
+    * @return String
+    */
+  def convertDegToDms(baseDegree: String): String = {
+    val dmsDegree: Double = baseDegree.toDouble
+
+    val degree: Double = Math.floor(dmsDegree)
+    val minute: Double = Math.floor((dmsDegree - degree) * 60)
+    val second: Double = Math.floor((((dmsDegree - degree) * 60) - minute) * 60)
+
+    val convertResult: String = degree.toInt.toString + "/" + minute.toInt.toString + "/" + second.toString
+
+    convertResult
```

Comment:
The variable `convertResult` is not needed.  Simply put the result expression instead of creating the result variable.

```scala
degree.toInt.toString + "/" + minute.toInt.toString + "/" + second.toString
```